tracer/src/Datadog.Trace/AppSec/Rcm/ConfigurationStatus.cs

```diff
@@ -61,9 +62,6 @@ internal record ConfigurationStatus
 
     internal IncomingUpdateStatus IncomingUpdateState { get; } = new();
 
-    /// <summary>
-    /// Gets or sets actions to take according to a waf result, these arent sent to the waf
-    /// </summary>
```

Comment:
They are still used out of this class for RC, so they cannot be set as private, but they do not contain the actions that we should take when the WAF returns the result because, from now, the WAF is already providing those actions.